odt-opt-in/initial ODT Opt-in JSON Contract.md

```diff
@@ -0,0 +1,155 @@
+# Open subshare JSON contract
+
+NOTES: every state checks memoMode - if memoMode is true, we cannot proceed.
+
+## PRELOADDATA
+To get things started, the client will send the initial PRELOADDATA request.
+```json
+{
+  "rgState": "PRELOADDATA",
+  "powerOnFileName": "BANNO.ODTOPTIN.V1.POW",
+  "userChrList": [
+    {"id": 1, "value": "1234567890S0010"},  [NOTE: 10-digit account+'S'+SHARE:ID]
+    {"id": 2, "value": ""},
+    {"id": 3, "value": ""},
+    {"id": 4, "value": ""},
+    {"id": 5, "value": ""}
+  ],
+  "userNumList": [],
+  "rgSession": 1
+}
+```
+
+Poweron response - when system is in Memo Mode:
+```json
+{
+  "memoMode": true,
+}
+```
+
+Poweron response - Config File read error:
+```json
+{
+  "errorCode":"501",
+  "loggingErrorMessage": "Error reading from config file:"+[detail]
+}
+```
+
+Poweron response - Invalid Account error:
+```json
+{
+   "errorCode":"502",
+   "loggingErrorMessage": "Invalid Account:"+[Invalid Account Type XX, Acct Warning XX, Acct Svc Code XX]
+}
+```
+
+Poweron response - If no eligible shares:
+```json
+{
+  "errorCode":"503",
+  "loggingErrorMessage": "No Eligible Shares"
+}
+```
+
+Poweron response - With eligible shares:
+```json
+{
+ "memoMode": false,
+ "results": {
+  "maxSharesExceeded": false, ['true' if the number of shares found exceeds processing capbilities (130 shares)]
+  "shareDetail": [{           [note: the first share returned will be the share the member]
+    "SID": "0000",            [currently has opened, provided it is eligible. All other shares will be in heirarchal order]
+    "name": "My Primary Share"
```

Comment:
missing a comma at the end of this line and the next. Same for the second example share

Reply:
corrected - waiting for clarification on other issues before posting update